sbndcode/Commissioning/HitDumper_module.cc

```diff
@@ -581,7 +581,7 @@ void Hitdumper::analyze(const art::Event& evt)
     _n_crt_strip_hits = crtStripHitVector.size();
   }
   else
-    std::cout << "Failed to get sbnd::crt::CRTStripHit data product." << std::endl;
+    std::cout << "Failed to get sbnd::crt::CRTStripHit data product (fCRTStripHitModuleLabel)." << std::endl;
```

Comment:
This will just print the word "fCRTStripHitModuleLabel" not the value of the object.

```suggestion
    std::cout << "Failed to get sbnd::crt::CRTStripHit data product ( " << fCRTStripHitModuleLabel << ")." << std::endl;
```